docs/topics/testing/tools.txt

```diff
@@ -1687,6 +1687,45 @@ your test suite.
             Person.objects.create(name="Aaron")
             Person.objects.create(name="Daniel")
 
+.. method:: TestCase.captureOnCommitCallbacks(self, *, using="default", execute=False)
```

Comment:
I moved this docs from ["Assertions"](https://docs.djangoproject.com/en/dev/topics/testing/tools/#assertions) to [`TestCase`](https://docs.djangoproject.com/en/dev/topics/testing/tools/#testcase).